{{ cookiecutter.project_slug }}/manage.dist.py

```diff
@@ -3,6 +3,9 @@
 import sys
 
 if __name__ == "__main__":
+    import environ
```

Comment:
@kaniak274 now it's loaded *both* in manage.py's and base.py - is that okay?

Reply:
It is. I.e. I'm reading `.env` file in manage.py files and i'm getting instance of the `environ` in the base.py with already read variables.

Reply:
But then, I _think_ that also WSGI runner should load env's too. Manage py is not used with gunicorn at all.